js/app.js

```diff
@@ -1,19 +1,63 @@
 /*-------------------------------- Constants --------------------------------*/
+const characters = ['❤️', '💰', '⚽', '💎', '🤑']
+const winningMessage =  `You have won`
+const winVoice = new Audio('Win.mp3')
+/*-------------------------------- variables--------------------------------*/
+ let char 
+ let wins = 0
 
+/*-------------------------------- Cached Element References--------------------------------*/
 
-
-/*-------------------------------- Variables --------------------------------*/
-let youWin, 
-
-
-
-/*-------------------------------- Constants Element References--------------------------------*/
-const machine
-
+var button = document.querySelector('.spin')
+var  win = document.getElementById('win')
+var winnings = document.getElementById('score')
 
 /*----------------------------- Event Listeners -----------------------------*/
-
-
-
-
-/*-------------------------------- Functions --------------------------------*/
\ No newline at end of file
+button.addEventListener('click', function(event){
+  return spin()
+})
+
+
+/*-------------------------------- Functions --------------------------------*/
+
+function getRandomNumber() {
+  var char = characters[Math.floor(Math.random() * characters.length)];
+  return char
+}
+getRandomNumber();
+function getElement(id) {
+  return document.getElementById(id);
+}
+
+function spin() {
+  const item1 = getElement('item1');
+  const item2 = getElement('item2');
+  const item3 = getElement('item3');
+
+  getRandomNumber();
+  const num1 = getRandomNumber();
+  const num2 = getRandomNumber();
+  const num3 = getRandomNumber();
+
+  
+item1.innerHTML = `${num1}`;
+  item2.innerHTML = `${num1}`;
+  item3.innerHTML = `${num3}`;
+
+   if (num1 === num3){
+    win.innerHTML = winningMessage    
+} if(num1 === num3){
+  winVoice.volume = .10
+  winVoice.play()
+}
+
+ else {
+  win.innerHTML = ''
+}
+  if (num1 == num3){
+    wins++
+    winnings.innerHTML = ` 💵Won ${wins}K $`
+  } else if(num1 !== num3){
+    winnings.innerHTML = ''
+  }
+}
```

Comment:
There's multiple indentation and spacing errors that make lines 43- 62 difficult to parse, but all of this code can be simplified down to a single `if ... else` statement. That would be helpful for 2 main reasons:
- fewer potential issues with indentation and spacing
- much less confusing to read, and understand what's going on here.

On lines 47, 49, and 57, you're checking for the same exact condition - if `num1` is equal to `num3`. There is no reason to do these things separately because they are not dependent on one another. Same thing with your `else` on line 54 and `else if` on line 60. Finally, you don't want to clear out the `winnings` when the player loses.

You should also do something to the player score when the player loses! Right now your game doesn't simulate slots very well since there's no possible way for the player to actually lose anything. 

These lines should also be in their own `render` function because their job is to display the state of the game to the user. Here's that function, with these code changes made:

```javascript
function render() {
  if (num1 === num3){
    winVoice.volume = .10
    winVoice.play()
    win.innerHTML = winningMessage
    wins++
  } else {
    win.innerHTML = ''
  }
  winnings.innerHTML = ` 💵Won ${wins}K $`
}
```